cli/tauri.js/bin/tauri.js

```diff
@@ -66,6 +71,12 @@ const tauri = function (command) {
     console.log(`Invalid command ${command}. Use one of ${cmds.join(', ')}.`)
   }
 }
+// notifying updates.
+const pkg = JSON.parse(fs.readFileSync(__dirname + '/../package.json')); 
```

Comment:
You already `require`d the `package.json` earlier, why are you reading it here as well?